### PR TITLE
Wolvic experimental devtools

### PR DESCRIPTION
**Pull request by tiagovignatti**
@zakharvoit @svillar PTAL.

### CODE REVIEW

File: wolvic/wolvic_main_parts.cc [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 #include "wolvic/wolvic_main_parts.h"
 
+#include "content/shell/browser/shell_devtools_manager_delegate.h"
```
Comment (zakharvoit) [inline wolvic/wolvic_main_parts.cc]: nit: alphabetically sort the includes

**Review by zakharvoit — COMMENTED** [review]
I couldn't make it work on Oculus via usb. Did you test it via usb or tcp?

File: wolvic/wolvic_main_parts.cc (hunk 1 above)
Comment (tiagovignatti) [reply wolvic/wolvic_main_parts.cc]: done.

**Comment by tiagovignatti** [comment]
> I couldn't make it work on Oculus via usb. Did you test it via usb or tcp?

I've tested via USB.

Copy-pasting here the commit message that describes what I've done to test (in case you've missed): 
I've tested by connecting my Android device (Samsung S21), launching Wolvic Chromium (using `noapi` variant) via Android Studio and then opening `chrome://inspect` in the host machine. I'm able to see Wolvic in the Remote Target list and connect into. I can't interact much with the DevTools' functionalities due the fact that the `noapi` variant doesn't render content. 

In the host machine, I have discover network targets as the Chromium default location at `localhost:9222`

image

**Review by zakharvoit — APPROVED** [review]

**Comment by tiagovignatti** [comment]
@jdapena could you PTAL? Thank you.

**Review by jdapena — APPROVED** [review]
We will want to eventually move to avoid using anything from content/shell. But that does not need to happen immediately, so we should we good with those changes.

Thanks!

**Pull request merged**